include/ideep/operators/conv.hpp

```diff
@@ -1838,20 +1838,25 @@ struct convolution_forward
     if (reorder_src) {
       auto expected_dst_desc = param.pd.dst_desc();
       tensor expected_dst;
-      // dst not init in FW or has same desc with expected desc.
-      if (dst.is_empty() || dst.get_desc() == expected_dst_desc) {
-        dst.reinit_if_possible(expected_dst_desc);
-        expected_dst = dst;
-      } else {
+      if (dst.is_empty() || dst.get_desc() != expected_dst_desc){
+        // If dst buffer are not given by user or user given dst buffer are not under expected format
+        // We need init a new one
         expected_dst.init(expected_dst_desc);
-        if (param.op_attr.has_op_kind(kind::sum)) {
+        if (!dst.is_empty() && param.op_attr.has_op_kind(kind::sum)) {
+          // We need copy the content of given buffer if matmul is fused with sum
```

Comment:
I guess you wanted to say "conv is fused with sum"?

Reply:
Ok It's now fixed.